backend/backend/appsettings.json

```diff
@@ -10,5 +10,9 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
+  "EmailConfiguration": {
+    "EmailAddress": "wedontbyte.team@gmail.com",
+    "EmailPassword": ""
```

Comment:
Where do we store password now? Maybe we should consider storing it as environmental variable?

Reply:
If I recall correctly we decided not to store the password in environmental varibles.
And because we do not want to show the password in github repo, we decided to remove it from appsettings.json
When you are working with the email sender locally, just paste in the password to appsettings